security-release-process-documentation/security-release-process.md

```diff
@@ -81,6 +81,10 @@ The initial Product Security Team will consist of volunteers subscribed to the p
 - Tim Allclair (**[@tallclair](https://github.com/tallclair)**) `<tallclair@google.com>` [4096R/0x5E6F2E2DA760AF51]
 - Jordan Liggitt (**[@liggitt](https://github.com/liggitt)**) `<jordan@liggitt.net>` [4096R/0x39928704103C7229]
 
+[Associate](#Associate) members include:
+
+- Jonathan Pulsifer (**[@jonpulsifer](https://github.com/jonpulsifer)**) `<jonathan@pulsifer.ca>`
```

Comment:
@jonpulsifer Is this the email address you want to use (pulled from your public github profile). Also, do you have a PGP key you want to add?